src/nvim/api/buffer.c

```diff
@@ -711,7 +711,7 @@ Integer nvim_buf_add_highlight(Buffer buffer,
 /// @param line_end End of range of lines to clear (exclusive)
 ///                 or -1 to clear to end of file.
 /// @param[out] err Details of an error that may have occurred
-void nvim_buf_clear_highlight(Buffer buffer,
+void nvim_buf_del_highlight(Buffer buffer,
```

Comment:
IMHO this is wrong, this does not "delete" a highlight in the way you "delete" a specific key from a dict, rather it clears _all_ highlights of a source from a specified region of the file (typically the entire file).

(Also why is the "add single comment" button greyed out as if you are not supposed to use it? I get two emails every time someone does any "review" operation... Sorry OT)